zyte_common_items/processors.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, List
+
+from lxml.html import HtmlElement
+from parsel import Selector, SelectorList
+from zyte_parsers import Breadcrumb as zp_Breadcrumb
+from zyte_parsers import extract_breadcrumbs
+
+from .items import Breadcrumb
+
+
+def breadcrumbs_processor(value: Any, page: Any) -> Any:
+    def _from_zp_breadcrumb(value: zp_Breadcrumb) -> Breadcrumb:
+        return Breadcrumb(name=value.name, url=value.url)
+
+    if isinstance(value, SelectorList) and len(value) > 0:
```

Comment:
Hm, either my comment disappeared, or I haven't posted it :) Shouldn't we return None if the SelectorList matches nothing? I think currently an empty SelectorList is going to be returned.